app/src/screens/ProposalDetail.js

```diff
@@ -43,13 +50,51 @@ function ProposalDetail({ proposal, onBack, requestToken }) {
     executed,
   } = proposal
 
-  const handleWithdraw = useCallback(() => {
-    api.withdrawAllFromProposal(id).toPromise()
-  }, [api])
+  const { stakes, threshold } = getStakesAndThreshold(proposal)
+  const conviction = getCurrentConviction(stakes, blockNumber, alpha)
+  const myStakes = stakes.filter(({ entity }) => entity === connectedAccount)
```

Comment:
We could use `addressEqual`

```suggestion
  const myStakes = stakes.filter(({ entity }) => addressEqual(entity, connectedAccount))
```